nav2_controller/src/nav2_controller.cpp

```diff
@@ -167,6 +171,11 @@ ControllerServer::on_configure(const rclcpp_lifecycle::State & state)
     rclcpp_node_, "follow_path",
     std::bind(&ControllerServer::computeControl, this));
 
+  // Set subscribtion to the speed limiting topic
+  speed_limit_sub_ = create_subscription<nav2_msgs::msg::SpeedLimit>(
+    speed_limit_topic_, rclcpp::QoS(rclcpp::KeepLast(1)).transient_local().reliable(),
```

Comment:
Why local transient / reliable? I think this (and the costmap layer) should both be basically system defaults - except the queue size perhaps. They come up at the same time so storing the messages seems unnecessary.

Reply:
Changed to default QoS (volatile) for publisher: https://github.com/ros-planning/navigation2/pull/2074/commits/3465c90415afd11418f98fc01bbc9386afa853ea#diff-d344e376ecbc8e5ecd5045731e13855c6d2a9a955cdaee9b31dddbbee7681466L88-R86 and subscriber: https://github.com/ros-planning/navigation2/pull/2074/commits/3465c90415afd11418f98fc01bbc9386afa853ea#diff-7ce5c619d2d43062d3d66eff15e01887a0343a5865dfff941c579a6a0696f791L176-R176. Also, changed for all tests in the same commit.